### PR TITLE
Added an alt text to silly story generator

### PR DESCRIPTION
**Pull request by winniebosy**
#### Summary

 - I have added an alt attribute to the content.

#### Motivation

 
   -  Adding the alt attribute will facilitate in enhancing accessibility.
   - In the event the image fails to load, the alternative text will be displayed. In addition, people using assistive technologies such as the screen readers will not be left out from accessing the content.

#### Supporting details

[Ensure no missing alt attributes](https://github.com/mdn/content/issues/19334)

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by bsmth — CHANGES_REQUESTED** [review]
Hi @winniebosy, thanks a lot for the addition. It would be great to keep the alt text __approx 100 characters__.

Could you please shorten this to roughly one sentence around 100 chars long?  This is better for accessibility (some screen readers have a 125 char hard cutoff) and this helps keep the summary easy to understand!

**Comment by winniebosy** [comment]
>No problem @bsmth .. On it. Thanks

**Review by bsmth — APPROVED** [review]
Thanks a lot @winniebosy 🙌🏻
